src/ImageService.Models.Dto/Responses/ImageDataResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ImageService.Models.Dto.Responses.User
+{
+    public class ImageDataResponse
```

Comment:
maybe better name is 'ImageResponse' or 'ImageInfo'? we often use word Indo for Dto models

Reply:
I think we can make this a separate task for all tables